packages/loader/container-loader/src/test/connectionStateHandler.spec.ts

```diff
@@ -81,6 +81,8 @@ describe("ConnectionStateHandler Tests", () => {
         );
     });
 
+    const quorum = (protocolOplHandler: IProtocolHandler): Quorum => protocolOplHandler.quorum as Quorum;
```

Comment:
I think you can keep ProtocolOpHandler  to have Quorum to avoid these casts. I.e. concrete classes expose concrete classes, and interfaces exposed interfaces.